tests/code/test_proposals.py

```diff
@@ -1,17 +1,54 @@
 import pytest
-from peewee import IntegrityError, DoesNotExist
-
+from flask import Flask, g
+from peewee import DoesNotExist
 from app.models.program import Program
+from app.models.course import Course
+from app.models.courseParticipant import CourseParticipant
+from app.models.courseInstructor import CourseInstructor
+from app.models.courseQuestion import CourseQuestion
 from app.models.user import User
-from app.logic.getSLInstructorTableData import getProposalData
-from app.controllers.admin.changeSLAction import withdrawCourse
+from app.logic.courseProposals import deleteProposal, getProposalData, authorized
```

Comment:
Not sure if this is just on my end, but I'm getting an error from this line about importing authorized from courseProposals.py.  Doesn't look like you use it in this test suite either.  Is it needed?